readme.md

```diff
@@ -25,7 +25,7 @@ Your Answer:
 Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
-```text
+```In brief, jQuery includes the javascript commands necessary to capture html or radio selections from the DOM and communicates this to the backend generating an asynchronous response that is ideally seamless to the user.
 
```

Comment:
:+1:

Pretty much. You'll also want to make sure you've set up an event listener that will trigger the AJAX request in the first place.